src/main/groovy/org/gradlefx/configuration/sdk/states/AbstractCreateSdkInstallLocationState.groovy

```diff
@@ -32,14 +32,14 @@ abstract class AbstractCreateSdkInstallLocationState implements SdkInitState {
     SdkInstallLocation installLocation
     String sdkFile
     protected String configName
-    Boolean isInstallationRequired
+    Boolean hasSDKDeclaredAsDependency
```

Comment:
this is just a small refactoring to make this part of the code more understandable.